Brokerages/Alpaca/AlpacaBrokerage.cs

```diff
@@ -107,6 +97,7 @@ public AlpacaBrokerage(IOrderProvider orderProvider, ISecurityProvider securityP
                 ApiEndpoint = tradingMode.Equals("paper") ? Environments.Paper.AlpacaTradingApi : Environments.Live.AlpacaTradingApi,
                 SecurityId = new SecretKey(accountKeyId, secretKey)
             });
+
             // api client for alpaca data
             _polygonDataClient = new PolygonDataClient(new PolygonDataClientConfiguration
```

Comment:
I think this `PolygonDataClient` is being used to download the history data, believe we might wanna consolidate it with the new `polygon` class?